Cargo.toml

```diff
@@ -14,3 +14,5 @@ members = [
 [dependencies]
 btf_types = { path = "btf_types" }
 btf_interp = { path = "btf_interp" }
+structopt = { version = "0.3", features = [ "paw" ] }
+paw = "1.0"
```

Comment:
The method I'd recommend these days is `clap` with `clap-derive`

Reply:
I have tried some alternatives such as clap, argh and docopt and since in the video you said that clap is more height weight, I tried structopt.
Looks like the most light weight option is argh, but there is no built in --version method